### ISSUE TITLE
Support copying over files not listed in the manifest

### ISSUE THREAD

**Issue by avi12** [extend-chrome/rollup-plugin-chrome-extension#62]
### Is your feature request related to a problem? Please describe.

In a normal circumstance, the developer would want a content script to be loaded as usual.
However, in my use-case, I need to be able to inject a whole script directly into a web page to use the JavaScript that lives on that page.
More specifically, I made a [YouTube Speed Control](https://github.com/avi12/youtube-speed-control) extension and I wish to replace its setup with Rollup, hopefully using this plugin.

### Describe the solution you'd like
Have the ability to copy over any JavaScript file, regardless if it's listed in the manifest, or let the developer manually list files that are required to be copied-over.

### Describe alternatives you've considered

I tried listing [that](https://github.com/avi12/youtube-speed-control/tree/master/src/scripts/yt-speed-control-script-to-inject.js) script in the content script array, and it does execute as usual, but since the script relies on a function that lives in the page itself, it won't really work, or at least not cleanly, as I constantly need to check for the function's existence.

### Additional context

In my use-case, I rely on [YouTube's `.setPlaybackRate()`](https://developers.google.com/youtube/iframe_api_reference#Playback_rate) function as it's the most reliable and clean way for my extension to work.
Basically, once the content script loads up, it sends a message to the background, which fetches the script-to-be-injected, and the content script injects that script, after which the script takes the wheel and changes the speed of the YouTube player according to the user's preference.

**Comment by jacksteamdev** [comment]
@avi12 This can be done using another Rollup plugin like [`rollup-plugin-copy2`](https://github.com/mentaljam/rollup-plugin-copy2) which adds additional files to the bundle.

`rollup-plugin-chrome-extension` deals mainly with parsing, modifying and emitting a Chrome extension's `manifest.json` file.
